dataland-frontend/tests/e2e/fixtures/eutaxonomy/non-financials/EuTaxonomyDataForNonFinancialsFixtures.ts

```diff
@@ -98,7 +98,7 @@ export class EuNonFinancialsGenerator extends Generator {
    */
   generateEuTaxonomyPerCashflowType(): EuTaxonomyDetailsPerCashFlowType {
     return {
-      totalAmount: this.randomDataPoint(generateCurrencyValue(), generateCurrencyCode()),
+      totalAmount: this.randomCurrencyDataPoint(generateCurrencyValue()),
```

Comment:
the generateCurrencyValue can be left out

Reply:
Also the same in SFDR

Reply:
Removed in both files